dts/x86/arduino_101.fixup

```diff
@@ -1,7 +1,5 @@
-#define CONFIG_UART_QMSI_0_BAUDRATE	INTEL_ARDUNIO101_UART_B0002000_CURRENT_SPEED
```

Comment:
ok, little weird, lets get the other things in order than we can figure this out.